### PR TITLE
[Trivial] Update cloud ssl configuration example

### PR DESCRIPTION
**Pull request by utku-caglayan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,6 +205,7 @@ hazelcast:
 Cloud SSL configuration:
 ```
 ssl:
+    servername: "hazelcast.cloud"
```
Comment (yuce) [inline README.md:208]: How about setting this by default if the cloud is enabled, so the user doesn't have to configure it?

File: config/config.go [hunk 2]
```diff
@@ -92,6 +92,9 @@ func mergeFlagsWithConfig(flags *GlobalFlagValues, config *Config) error {
 	if flags.Token != "" {
 		config.Hazelcast.Cluster.Cloud.Token = strings.TrimSpace(flags.Token)
 		config.Hazelcast.Cluster.Cloud.Enabled = true
+		if config.SSL.ServerName == "" && !config.SSL.InsecureSkipVerify {
```
Comment (yuce) [inline config/config.go:95]: I think it's OK to not check for `InsecureSkipVerify`. Setting server name shouldn't affect that one.

File: config/config.go [hunk 3]
```diff
@@ -154,6 +151,11 @@ func DefaultConfigPath() string {
 }
 
 func updateConfigWithSSL(config *hazelcast.Config, sslc *SSLConfig) error {
+	var emptySSLConfig SSLConfig
+	if *sslc == emptySSLConfig {
```
Comment (yuce) [inline config/config.go:155]: How about: `*sslc == SSLConfig{}`

**Review by yuce — APPROVED** [review]
Looks good!

**Review by alisengul53 — APPROVED** [review]

**Pull request merged**